frontend/packages/dev-console/src/utils/imagestream-utils.ts

```diff
@@ -174,35 +174,14 @@ export const getSortedTags = (imageStream: K8sResourceKind) => {
       })
     : [];
 };
-export const getImageStreamTags = (imageStreams: K8sResourceKind[], name: string, ns: string) => {
-  const imageStream = _.find(imageStreams, (img) => {
-    return img.metadata.name === name && img.metadata.namespace === ns;
-  });
-  const sortedTags =
-    name && imageStream && !_.isEmpty(imageStream) ? getSortedTags(imageStream) : [];
+export const getImageStreamTags = (imageStream: K8sResourceKind | {}) => {
```

Comment:
Can `imageStream` be anything but a `K8sResourceKind`?
```suggestion
export const getImageStreamTags = (imageStream: K8sResourceKind) => {
```
Is there a specific reason you want to allow an empty object as a valid type?

Reply:
No Imagestream should be an K8sResourceKind only. I have removed this and now type casting the argument to `K8sResourceKind` from the caller function. Thanks Andrew i missed to change it.